src/main/scala/com/mozilla/telemetry/sinks/HttpSink.scala

```diff
@@ -117,8 +117,8 @@ abstract class HttpSink[T]() extends ForeachWriter[T] {
       case _ if successCodes.contains(code) => // pass; our work here is done.
       case _ if nextTry < maxAttempts && retryCodes.contains(code) => attempt(request, nextTry)
       case _ =>
-        val url = request.url + "?" + request.params.map{ case(k, v) => s"$k=$v" }.mkString("&")
-        log.warn(s"Failed request: $url, last status code: $code")
+        val body = response.map(": " + _.body).getOrElse("")
+        log.warn(s"Dropping request that failed with last status code `$code' $body")
```

Comment:
We previously were logging the entire payload, including the api_key. Especially with the batches of thousands of events, this leads to somewhat ridiculous output.

So, we replace this warning with just the body of the error message that's returned (which is a JSON blob describing what happened in Amplitude's case).